### DISCUSSION TITLE
Add Varlink to XDG DESKTOP PORTAL

### DISCUSSION THREAD

**Discussion by tgmatos** [flatpak/xdg-desktop-portal#1449] [Ideas]
### Expected Behavior

When dealing with alternative implementations of the Flatpak launcher a second dbus daemon overhead (xdg-dbus-proxy) limits the freedom of implementation. Varlink specs can give more freedom to approach the implementation. This feature has been asked before: https://blogs.gnome.org/mclasen/2019/10/03/some-flatpak-updates/#comment-20194

### Current Behavior

Currently Flatpak doesn't have support for Varlink specs.

### Anything else we should know?

_(Didn't find any flag to ask for this feature, so sorry to do it using a bug report.)_

**Comment by ryanabx** [comment]
Varlink has been adopted by systemd in swaths of their codebase, and it being simpler and introspectable without special tools is a nice bonus. It's also bus-less, which I like.
